### PR TITLE
Marks users of the system tenants with a permission, even if they are in spy mode

### PR DESCRIPTION
**Pull request by meg1502**

### CODE REVIEW

File: src/main/java/sirius/biz/tenants/TenantUserManager.java [hunk 1]
```diff
@@ -82,6 +82,15 @@
      */
     public static final String PERMISSION_SYSTEM_TENANT_MEMBER = "flag-system-tenant-member";
 
+    /**
+     * This flag permission is granted to <b>all users</b> which originally belong to the system tenant.
+     * <p>
+     * The id of the system tenant can be set in the scope config. The system tenant usually is the administrative
+     * company which owns / runs the system.
+     * Unlike {@link #PERMISSION_SYSTEM_TENANT_MEMBER}, this flag is kept when when the user either has taken control over another tenant or uses account.
```
Comment (sabieber) [inline src/main/java/sirius/biz/tenants/TenantUserManager.java]:
```suggestion
     * Unlike {@link #PERMISSION_SYSTEM_TENANT_MEMBER}, this flag is kept when the user either has taken control over another tenant or uses account.
```

Comment (welzfabian) [inline src/main/java/sirius/biz/tenants/TenantUserManager.java]:
```suggestion
     * Unlike {@link #PERMISSION_SYSTEM_TENANT_MEMBER}, this flag is kept when when the user either has taken control over another tenant or user account.
```

Comment (sabieber) [reply src/main/java/sirius/biz/tenants/TenantUserManager.java]: please clarify the last part after the `or`

Comment (welzfabian) [reply src/main/java/sirius/biz/tenants/TenantUserManager.java]: should probably be "user account" instead of "uses account" ?

**Pull request merged**